bin/compile

```diff
@@ -32,5 +32,9 @@ echo "-----> gem install bundler"
 gem install bundler
 echo "-----> bundle install"
 bundle install -j4 --deployment
+echo "-----> npm install --only=dev"
+npm install --only=dev
```

Comment:
@cgc are you sure about *only* devDependencies? We want to bundle them up I think...